contrib/notification-history.sh

```diff
@@ -1,47 +1,55 @@
-#!/bin/bash
-
-history_json=$(dunstctl history)
-
-history_length=$(echo $history_json|jq -r '.data[] | length')
-
-options=""
-
-for iter in $(seq $history_length); do
-    i=$((iter-1))
-    application_name=$(echo $history_json|jq -r .data[][$i].appname.data)
-    notification_summary=$(echo $history_json|jq -r .data[][$i].summary.data)
-    notification_timestamp=$(echo $history_json|jq -r .data[][$i].timestamp.data)
+#!/bin/sh
```

Comment:
KIM: You've changed shells here. This might not be the way things go. You might not have seen a problem, if on your system `/bin/sh` is linked to bash.

Reply:
No, I wrote it deliberately only using POSIX features. I had and have `/bin/sh` linked to `dash`.